test/jdk/sun/java2d/pipe/InterpolationQualityTest.java

```diff
@@ -29,11 +29,11 @@
  * image via rendering hints for default, xrender and opengl pipelines.
  *
  * @author Vadim.Pakhnushev@oracle.com
```

Comment:
I think we are supposed to remove the author, if present historically, if we are making changes to any test.
rest looks ok.